data/json/items/comestibles/brewing.json

```diff
@@ -54,39 +102,40 @@
         "symbol" : "~",
         "nutrition" : 2,
         "quench" : 4,
-        "fun" : -15,
+        "fun" : -5,
         "price" : 0,
         "volume" : 1,
         "charges" : 7,
         "phase" : "liquid",
         "comestible_type" : "DRINK",
         "brewable" : {
             "time" : 7200,
-            "result" : "rum"
+            "result" : "wash_rum"
```

Comment:
`rum_wash` could be a better alternative. We seem to use both forms in the project but I think the preference for new items should be the most specific part first as this helps with ordering?